c++/src/kj/compat/http.c++

```diff
@@ -2411,15 +2419,24 @@ public:
 
     kj::Array<byte> message;           // space to allocate
     byte* payloadTarget;               // location into which to read payload (size is payloadLen)
+    kj::Maybe<size_t> maxMessageSize;  // maximum message size (maxSize from first `receive()` call)
```

Comment:
Bikeshed: `originalMaxSize` would make more sense to me.

Reply:
https://github.com/capnproto/capnproto/pull/1505/commits/d1fddc92daf46d4ee4813eb50005597f7fbb53d3